docs/root/version_history/current.rst

```diff
@@ -25,6 +25,7 @@ Bug Fixes
 * eds: fix the eds cluster update by allowing update on the locality of the cluster endpoints. This behavioral change can be temporarily reverted by setting runtime guard ``envoy.reloadable_features.support_locality_update_on_eds_cluster_endpoints`` to false.
 * tls: fix a bug while matching a certificate SAN with an exact value in ``match_typed_subject_alt_names`` of a listener where wildcard ``*`` character is not the only character of the dns label. Example, ``baz*.example.net`` and ``*baz.example.net`` and ``b*z.example.net`` will match ``baz1.example.net`` and ``foobaz.example.net`` and ``buzz.example.net``, respectively.
 * xray: fix the AWS X-Ray tracer extension to not sample the trace if ``sampled=`` keyword is not present in the header ``x-amzn-trace-id``.
+* outlier_detection: max_ejection_time_jitter configuration added to allow adding a random value to the ejection time to prevent 'thundering herd' scenarios. Defaults to 0 so as to not break existing deployments.
```

Comment:
Can you ref link to the new field?

Reply:
Added the link

Reply:
Can you please clarify what is meant by check CI? (this is my first contribution to Envoy)

Reply:
Down at the conversation page there is a failure in `envoy-presubmit (precheck format_pre)`. Pressing the `Details` link it will lead you to the CI results and eventually to this [link](https://dev.azure.com/cncf/envoy/_build/results?buildId=101138&view=logs&jobId=c5dd2866-6ab3-5f3c-3a44-4cef0ec909b5&j=c5dd2866-6ab3-5f3c-3a44-4cef0ec909b5&t=a9eb66d6-8944-5769-b3f7-476949dadcb8).

You should be able to reproduce the issue locally by running `ci/run_envoy_docker.sh 'ci/do_ci.sh format_pre'`.

Reply:
CI tests are now passing

Reply:
@adisuissa Is there a slack channel for envoy developers?

Reply:
Yeah, #envoy-dev at https://envoyproxy.slack.com/

Reply:
Thanks, can I please get an invite to this channel? My email is asheryer@amazon.com